binr/rax2/rax2.c

```diff
@@ -29,39 +29,39 @@ static int format_output(char mode, const char *s) {
 	}
 	switch (mode) {
 	case 'I':
-		printf ("%" PFMT64d "\n", n);
+		printf ("%" PFMT64d "%c", n, newline);
```

Comment:
This makes it more useful, for example if someone wants to get base64 of a number : 
```
sajjad@Sajjads-MacBook-Pro:~/Desktop$./x/bin/rax2 -l 0x12 | ./x/bin/rax2 -E
MTg=
```
```
sajjad@Sajjads-MacBook-Pro:~/Desktop$./x/bin/rax2 0x12 | ./x/bin/rax2 -E
MTgK
```